test/quicksiler-config-client.test.js

```diff
@@ -18,4 +18,25 @@ describe('QuicksilverConfigClient', () => {
       mbConfigPath: process.env.MB_CONFIG_PATH,
     });
   }
+
+
+  // Constructor.
+  describe('constructor', () => {
+    // Test new instance.
+    it('without options should throw a TypeError', () => {
+      (() => new QuicksilverConfigClient()).should.throw(TypeError);
+    });
+
+    // Check API base URL.
+    it('base URL option should be set', () => {
+      process.env.MB_CONFIG_PATH.should.be.not.empty();
```

Comment:
The repo that contains the settings is called "`messagebroker-config`" and the file is called "`mb_config.json`". I was thinking the config var `MB_CONFIG_PATH` should reflect these names.  Ideally this will all change as things are renamed.

I'm happy to use "`Q_CONFIG_PATH` if you think it's that's more inline with the current naming conventions?